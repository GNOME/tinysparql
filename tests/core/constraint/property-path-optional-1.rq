CONSTRAINT GRAPH <http://example.com/A>
SELECT ?s {
  ?s rdf:type? example:A
}
ORDER BY ?s
