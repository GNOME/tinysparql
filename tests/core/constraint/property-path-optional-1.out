"http://example/A"
"http://example/a"
